install/no-https

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+if [[ $# != 2 ]]
+then
+		echo "Usage: "
+		echo "	$0 userName fullURLToSite"
+		exit 1
+fi
+
+userName=$1
+FQDN=$2
+configFile=/etc/nginx/sites-available/$FQDN
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
+
+# Create logfolder+configfile
+mkdir -p /var/log/nginx/$FQDN
+touch $configFile
```

Comment:
Why is this touch line needed? Tried running the cat command below without it and it seemed to create the file just fine without it.